obs-websocket-dotnet/OBSWebsocket_Requests.cs

```diff
@@ -1744,7 +1744,7 @@ public List<JObject> GetGroupSceneItemList(string sceneName)
             };
 
             var response = SendRequest(nameof(GetGroupSceneItemList), request);
-            return JsonConvert.DeserializeObject<List<JObject>>((string)response["sceneItems"]);
```

Comment:
All other functions are in the format of `(type)resonse["keyName"]` is there a reason we're changing it here?

Reply:
The deserialization does not like the direct cast here, causing the request to hang. Now that you have me thinking, maybe changing List to something like a JArray could work as well.

Reply:
When I was testing with my code, it was failing on line 1747 saying it couldn't convert from Array to String but I changed it to this and got it to work:

`return JsonConvert.DeserializeObject<List<SceneItemDetails>>(response["sceneItems"].ToString());`

You'll notice I also changed the overall function to return as SceneItemDetails like it was before but apparently for whatever reason the (string) isn't allowed whereas .ToString() is fine.